pt-BR/mail/chrome/messenger/msgAccountCentral.dtd

```diff
@@ -1,20 +1,20 @@
-<!ENTITY emailSectionHdr.label        "Mensagens"> 
-<!ENTITY readMsgsLink.label           "Ler mensagens"> 
-<!ENTITY composeMsgLink.label         "Criar uma nova mensagem"> 
+<!ENTITY emailSectionHdr.label        "Mensagens">
+<!ENTITY readMsgsLink.label           "Ler mensagens">
+<!ENTITY composeMsgLink.label         "Criar uma nova mensagem">
 
-<!ENTITY newsSectionHdr.label         "Newsgroups"> 
-<!ENTITY subscribeNewsLink.label      "Gerenciar inscrições de newsgroups"> 
+<!ENTITY newsSectionHdr.label         "Newsgroups">
+<!ENTITY subscribeNewsLink.label      "Gerenciar inscrições de newsgroups">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY subscribeNewsLink.label      "Gerenciar inscrições em grupos de notícias">